docs/data/sql_functions.yml

```diff
@@ -1178,6 +1178,24 @@ valueconstruction:
       ```sql
       f(columns => DESCRIPTOR(`col1`, `col2`), on_time => DESCRIPTOR(`ts`))
       ```
+  - sql: OBJECT_OF(className, [key, value [, key, value , ...]])
+    table: objectOf(STRING, Object...)
+    description: |
+      Creates a structured object from a list of key-value pairs.
+
+      The function creates an instance of a structured type identified by the given class name.
+      The structured type is created by providing alternating key-value pairs where keys must be
+      string literals and values can be arbitrary expressions.
+
+      Users are responsible for providing a valid fully qualified class name that exists
+      in the classpath. The class name should follow Java naming conventions (e.g., 'com.example.User').
```

Comment:
is this the user classpath?

Reply:
Yes, this is also mentioned in the StructuredType JavaDocs
https://github.com/apache/flink/blob/master/flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/logical/StructuredType.java#L85-L88